### PR TITLE
Add instructions to start Postgresql and Redis servers

### PR DESCRIPTION
**Pull request by akshayKhot**
I tried to set up the local development environment using the setup instructions. The `bin/setup` script failed as the database and Redis servers weren't running. This pull request adds the instructions to start those before running the script. 

Note: This is my first time contributing to any open source project. Please let me know if I missed something or anything that needs to be corrected. 

### Pull request checklist

- [x] Your code contains tests relevant for code you modified
- [x] You have linted and tested the project with `bin/check`
- [x] You have added significant changes and product updates to the [changelog](CHANGELOG.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,18 @@ You will also need Google Chrome installed to run the system tests.
 
 ### Initial setup
 
+Start Postgresql server.
```
Comment (joemasilotti) [inline README.md:42]:
```suggestion
Start PostgreSQL server.
```

**Comment by joemasilotti** [comment]
Ahhh, got it! Great catch.

I made one suggestion to the PR. Along with that, can you remove the "Next steps" section at the bottom of `bin/setup`?

**Comment by akshayKhot** [comment]
Sounds good, I have made the suggested corrections. Thanks for the feedback.

**Comment by joemasilotti** [comment]
Look great – thank you for your contribution!

**Pull request merged**